lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -36,7 +36,7 @@ def self.miq_log_workspace(obj, _inputs)
     end
 
     def self.miq_send_email(_obj, inputs)
-      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"], :cc => inputs["cc"], :bcc => inputs["bcc"], :content_type => inputs["content_type"])
+      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"], {:cc => inputs["cc"], :bcc => inputs["bcc"], :content_type => inputs["content_type"]})
```

Comment:
I needed to change this as the callee's interface was changed to support ruby 2.6-3.0+ so it no longer accepts keyword arguments directly.  Notice, this is because `execute_with_user` below now captures any kwargs as a hash and appends that to the args so no kwargs are sent to the callee in `MiqAeServiceMethods`.

Reply:
A little surprised that you needed this change.
Seems it would have been converted to a hash automatically in ruby

Reply:
On ruby 2/3, the `send_email` interface is 5 arguments via this PR.  This call was passing only 4 on ruby 3 as kwargs are not an "argument".

Reply:
I can try to avoid changing the interface of `send_email` and see if I can "fix" the inputs to work with ruby 2/3.  I thought I tried it but now that there are tests, we can experiment.

Reply:
This is now stale.  I figured out a way to fix send_email to accept kwargs or an options hash and treat it differently since it's really the only one doing kwargs.